.github/workflows/furnishings-ci.yml

```diff
@@ -82,6 +82,14 @@ jobs:
         id: test
         run: |
           make test
+      - name: Upload coverage to Codecov
+        uses: codecov/codecov-action@v4
+        with:
+          file: ./jobs/furnishings/coverage.xml
```

Comment:
Seems furnishings & involuntary dissolutions jobs don't have this file in place. Can you add it?

Reply:
Thank you for mentioning this. However, coverage.xml is in the gitignore, so they won't be push to repos anyway 
![image](https://github.com/bcgov/lear/assets/169101247/9dfcbdde-d505-4d0b-9637-417f01deee5a)

This inspired me to check the `codecov.yaml`. @argush3 will that be a good idea to add flags under coverage and adding  paths for Entity BN, Entity Digital Credentials, Furnishings Job, Involuntary Dissolutions Job to `codecov.yaml` file?
![image](https://github.com/bcgov/lear/assets/169101247/a61c898f-a91c-4acc-9de8-eb3055356c27)

Reply:
yes, I think that woudl be a good idea to add these flags to codegov.yaml.  I would remove datareset too